pdns/mastercommunicator.cc

```diff
@@ -58,15 +58,16 @@ void CommunicatorClass::queueNotifyDomain(const DomainInfo& di, UeberBackend* B)
       nsset.insert(getRR<NSRecordContent>(rr.dr)->getNS().toString());
 
     for(const auto & j : nsset) {
-      vector<string> nsips=fns.lookup(DNSName(j), B);
+      auto dnsj = DNSName(j);
```

Comment:
when you're here anyway. Maybe rename `j` to something better?

Reply:
I did something even better :)